### PR TITLE
Update docs for autodeployment routes

### LINKED ISSUE #2546: https://dev.api.telescope.cdot.systems/deploy/status is 404
https://dev.api.telescope.cdot.systems:4000/status works but https://dev.api.telescope.cdot.systems/deploy/status gives 404. Do I have the URL wrong, or is this broken?

cc @AmasiaNalbandian, @manekenpix

### PR DESCRIPTION
**Pull request by dbelokon**
## Issue This PR Addresses

Fixes #2546 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [X] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description

This PR updates the documentation detailing the new routes for the autodeployment server.

## Steps to test the PR

Nothing to test here... 👀 

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [X] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by humphd — CHANGES_REQUESTED** [review]
Let's also include links to the new Build Log Dashboard page, where you can see these logs in real-time.

**Review by tpmai22 — CHANGES_REQUESTED** [review]
I check the links for both `staging` and `production` but they seem to be broken

**Comment by dbelokon** [comment]
> I check the links for both `staging` and `production` but they seem to be broken

I think this might be a separate issue. Also, I just checked the links to both `staging` and `production` and both of them give a response, so I think they are the actual links.

File: tools/autodeployment/README.md [hunk 1]
```diff
@@ -17,10 +17,23 @@ auto deployment server:
 - `GET /deploy/status`: returns JSON data about the current status of the build.
 - `GET /deploy/log`: returns a real-time stream of current build log output, if any.
 
-On staging you can go to http://dev.telescope.cdot.systems/status and
-http://dev.telescope.cdot.systems/log, and on production you can use
-http://telescope.cdot.systems/status and http://telescope.cdot.systems/log.
+On staging you can go to:
+
+- https://dev.telescope.cdot.systems/deploy/status to check the deployment status,
+- https://dev.telescope.cdot.systems/deploy/log to check the deployment log,
```
Comment (humphd) [inline tools/autodeployment/README.md:23]: FYI, there are two more: https://dev.telescope.cdot.systems/deploy/log/previous and https://dev.telescope.cdot.systems/deploy/log/current (https://dev.telescope.cdot.systems/deploy/log redirects to https://dev.telescope.cdot.systems/deploy/log/current).

File: tools/autodeployment/README.md [hunk 2]
```diff
@@ -17,10 +17,23 @@ auto deployment server:
...
+
+and on production you can use
+
+- https://telescope.cdot.systems/deploy/status to check the deployment status,
+- https://telescope.cdot.systems/deploy/log to check the deployment log.
```
Comment (humphd) [inline tools/autodeployment/README.md:28]: Same for this one.

**Review by humphd — CHANGES_REQUESTED** [review]
Looks good, one thing to add.

**Review by tpmai22 — APPROVED** [review]
Wish you add this sooner when I was struggled finding my way in `autodeployment` !

**Comment by tpmai22** [comment]
I think we are good to go with this @humphd

**Review by humphd — APPROVED** [review]

**Pull request merged**